tests/core/parameter_validation/test_parameter_validation.py

```diff
@@ -49,11 +49,30 @@ def test_filesystem_hierarchy():
     path = os.path.join(BASE_DIR, 'filesystem_hierarchy')
     parameters = ParameterNode('', directory_path = path)
     parameters_at_instant = parameters('2016-01-01')
-    assert parameters_at_instant.node1.param == 1.0
+    assert_equals(parameters_at_instant.taxes.rate, 0.22)
 
 
 def test_yaml_hierarchy():
     path = os.path.join(BASE_DIR, 'yaml_hierarchy')
     parameters = ParameterNode('', directory_path = path)
     parameters_at_instant = parameters('2016-01-01')
-    assert parameters_at_instant.node1.param == 1.0
+    assert_equals(parameters_at_instant.taxes.rate, 0.22)
+
+
+def test_parameters_metadata():
+    path = os.path.join(BASE_DIR, 'filesystem_hierarchy')
+    parameters = ParameterNode('', directory_path = path)
+    assert_equals(parameters.taxes.rate.reference, ['http://legifrance.fr/taxes/rate'])
+    assert_equals(parameters.taxes.rate.unit, '/1')
+    assert_equals(parameters.taxes.rate.values_list[0].reference, ['http://legifrance.fr/taxes/rate/2015-12'])
+    assert_equals(parameters.taxes.rate.values_list[0].unit, '/1')
+
+
```

Comment:
I see a test for `parameter`, `parameter_node`, and `parameter_at_instant` and not for `scale`. Is that intentional ?